source/apiVolontaria/volunteer/views.py

```diff
@@ -1,9 +1,10 @@
 
 from rest_framework import generics, status
 from rest_framework.response import Response
-from rest_framework.permissions import IsAuthenticated
+from rest_framework.permissions import IsAuthenticated, DjangoObjectPermissions
 
-from .permissions import IsOwnerOrReadOnly
+from volunteer.models import Participation, Cell, Event
```

Comment:
We already have an import `from . import models, serializers` two line after this new one.

Reply:
Good point